chopper.py

```diff
@@ -119,62 +118,81 @@ def parse_args() -> dict[str, Any]:
         default=0,
     )
 
-    args = vars(parser.parse_args())
+    config = parser.parse_args()
 
-    if not args["input_path"].exists():
-        parser.error(
-            "Invalid input_path. Please specify an existing file or directory."
-        )
+    for path in config.input_paths:
+        if not path.exists():
+            parser.error(
+                "Invalid input_paths. Please specify existing file(s) and/or directory (directories)."
+            )
 
-    if not args["output_directory"].exists():
-        os.makedirs(args["output_directory"], exist_ok=True)
+    if not config.output_directory.exists():
+        config.output_directory.mkdir(exist_ok=True)
 
-    actions = ["columns", "rows", "equal", "shuffles"]
-    if not args.keys() & actions:
+    actions = {"columns", "rows", "equal", "shuffles"}
+    if len(set(config.__dict__.keys()).intersection(actions)) == 0:
         parser.error(
             "You must specify at least one action (columns, rows, equal, shuffles)."
         )
 
-    return args
+    return config
+
 
+def get_offsets(file: TextIOWrapper) -> list[int]:
+    pos = 0
+    row_offsets = []
 
-def shuffle_file(filepath: Path, shuffles: int, config: Config) -> list[str]:
+    for line in file:
+        pos += len(line)
+        row_offsets.append(pos)
+
+    return row_offsets
```

Comment:
In my testing, there was no major difference between the chunked version of this and the by-line version, so went with the simpler version. Both are quite fast to generate the positions.

cc @camgunz